routemaster.rb

```diff
@@ -2,4 +2,30 @@ module Routemaster
   def self.now
     (Time.now.utc.to_f * 1e3).to_i
   end
+
+  DEFAULTS = {
+    redis_pool_size: 1
+  }.freeze
```

Comment:
Because you wouldn't want to accidentally modify the defaults hash? It's not required. I can remove if it's confusing.